lib/src/analyzers/lint_analyzer/rules/rules_list/format_comment/format_comment_rule.dart

```diff
@@ -24,15 +24,22 @@ class FormatCommentRule extends CommonRule {
   static const _warning = 'Prefer formatting comments like sentences.';
 
   FormatCommentRule([Map<String, Object> config = const {}])
-      : super(
+      : _ignoredPatterns = List<String>.from(
+          config['ignored-patterns'] as List? ?? const <String>[],
+        ).map((stringPattern) => RegExp(stringPattern)),
+        super(
           id: ruleId,
           severity: readSeverity(config, Severity.style),
           excludes: readExcludes(config),
         );
 
+  /// The patterns to ignore. They are used to ignore and not lint comments that
+  /// match at least one of them.
+  final Iterable<RegExp> _ignoredPatterns;
```

Comment:
Please declare this variable before the constructor.

Reply:
I moved the variables in this commit: https://github.com/dart-code-checker/dart-code-metrics/pull/828/commits/5ef888526b251c5dbacaf2e7c2689ab933ed06ad